educator/src/Dscp/Educator/Web/Logic.hs

```diff
@@ -19,3 +31,22 @@ commonGetProofs filters = do
         | (mtree, indicedTxs) <- rawProofs
         , let bpiTxs = map snd indicedTxs
         ]
+
+getEducatorStatus
+    :: MonadEducatorWeb ctx m
+    => m EducatorInfo
+getEducatorStatus = do
+    sk <- ourSecretKeyData @EducatorNode
+    let address = skAddress sk
+
+    -- TODO [DSCP-367]: to replace with one-liner
+    accounts <- readingSDLock $ do
+        blockAccount <- runSdReadM $ getAccountMaybe address
+        poolAccount  <- runSdReadM $ getMempoolAccountMaybe address
+        return $ fromMaybe def <$>
+                 BlocksOrMempool{ bmConfirmed = blockAccount, bmTotal = poolAccount }
+
+    return EducatorInfo
+        { eiAddress = address
+        , eiBalances = leftToPanic . coinFromInteger . aBalance <$> accounts
```

Comment:
Either use `unsafeMkCoin`, or throw an error inside a monad instead.
Another question is: why the heck `aBalance` in `Account` is not a `Coin` already?...

Reply:
I guess we don't want `Account` to keep `Coin` because `< maxBound` checks are completely unnecessary in expansion and validation where `Account` is primarily used. Actually, this check is an extra here too, so `unsafeMkCoin` looks like a nice option.

Although, what do you think about this way of resolving the inconsistency (bounded in some places, and not bounded in others) in `Coin` semantics?
https://issues.serokell.io/issue/DSCP-363#comment=96-11517